userguide/services.rst

```diff
@@ -1955,13 +1955,12 @@ UPS Configuration screen.
    |                               |                | for more details about configuring multiple systems with a single UPS.                                                 |
    |                               |                |                                                                                                                        |
    +-------------------------------+----------------+------------------------------------------------------------------------------------------------------------------------+
-   | Port / Hostname / Remote Port | drop-down menu | :guilabel:`Port`: Enter the serial or USB port the UPS is connected to (see :ref:`NOTE <UPS USB>`).                    |
+   | Port or Hostname              | drop-down menu | Enter the serial or USB port the UPS is using (see :ref:`NOTE <UPS USB>`).                                             |
```

Comment:
"is using" is kind of ambiguous.  Not clear on how it is using it.  Better is to be specific, but let's still avoid that split infinitive:
"Enter the serial or USB port connected to the UPS (see..."